tap_google_sheets/client.py

```diff
@@ -140,6 +142,12 @@ def __init__(self,
         self.__expires = None
         self.__session = requests.Session()
         self.base_url = None
+        # if request_timeout is other than 0,"0" or "" then use request_timeout
+        if request_timeout and float(request_timeout):
+            request_timeout = float(request_timeout)
+        else: # If value is 0,"0" or "" then set default to 300 seconds.
+            request_timeout = REQUEST_TIMEOUT
+        self.request_timeout = request_timeout
 
 
     def __enter__(self):
```

Comment:
Can we put backoff here and remove it from the get_access_token? so it will call for 5 times only instead of 25 times.

Reply:
Also need to add the ConnectionError here.

Reply:
> Also need to add the ConnectionError here so add like below:
> 
> ```
> @backoff.on_exception(backoff.constant,
>                           (Timeout,ConnectionError),
>                           max_tries=5,
>                           interval=10,
>                           jitter=None)
> ```
added the backoff here and the jitter